src/main/drivers/rx/expresslrs_driver.c

```diff
@@ -175,45 +162,21 @@ bool expressLrsTimerIsRunning(void)
 
 void expressLrsTimerStop(void)
 {
-#ifdef USE_HAL_DRIVER
-    LL_TIM_DisableIT_UPDATE(timer);
-    LL_TIM_DisableCounter(timer);
-    LL_TIM_SetCounter(timer, 0);
-#else
-    TIM_ITConfig(timer, TIM_IT_Update, DISABLE);
-    TIM_Cmd(timer, DISABLE);
-    TIM_SetCounter(timer, 0);
-#endif
+    timerDisable(timer);
+    timerSetCounter(timer, 0);
     timerState.running = false;
 }
 
 void expressLrsTimerResume(void)
 {
     timerState.tickTock = TOCK;
 
-#ifdef USE_HAL_DRIVER
-    LL_TIM_SetAutoReload(timer, (timerState.intervalUs / TICK_TOCK_COUNT));
-    LL_TIM_SetCounter(timer, 0);
-
-    LL_TIM_ClearFlag_UPDATE(timer);
-    LL_TIM_EnableIT_UPDATE(timer);
-#else
-    TIM_SetAutoreload(timer, (timerState.intervalUs / TICK_TOCK_COUNT));
-    TIM_SetCounter(timer, 0);
-
-    TIM_ClearFlag(timer, TIM_FLAG_Update);
-    TIM_ITConfig(timer, TIM_IT_Update, ENABLE);
-#endif
+    timerSetPeriod(timer, (timerState.intervalUs / TICK_TOCK_COUNT) - 1);
```

Comment:
Why the `-1`?

Reply:
@blckmn : ?

Reply:
Good question - this is been there for some time...

Reply:
I don't see - 1 in old code here

Reply:
Huh?

![image](https://github.com/user-attachments/assets/4e018884-dbce-49ef-af5c-84886616c2a7)